swap_meet/vendor.py

```diff
@@ -1,2 +1,61 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        inventory = inventory if inventory is not None else []
+        self.inventory = inventory 
+    
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item 
+        else:
+            return False 
+
+    def get_by_category(self,category):
+        items_list = []
+        for item in self.inventory:
+            if item.category == category:
+                items_list.append(item)
+        
+        return items_list 
+    
+    def swap_items(self, friend, my_item, friend_item):
+        if my_item not in self.inventory or friend_item not in friend.inventory:
+            return False
+        self.add(friend_item)
+        friend.add(my_item)
+        self.remove(my_item)
+        friend.remove(friend_item)
+
+        return True 
+    
+    def swap_first_item(self, friend):
+        if len (self.inventory) == 0 or len (friend.inventory) == 0:
```

Comment:
Nice guard clause. A more pythonic way to write this conditional is to check for it's falseyness

Reply:
So, how would that syntax look?   Would you expect:

**if not self.inventory and not friend.inventory**